Dockerfile

```diff
@@ -101,7 +92,8 @@ ENV PATH $PATH:/opt/elixir-${ELIXIR_VERSION}/bin
 
 # PHP Composer
 
-RUN apt-get update && apt-get install -y php-cli php-mbstring && apt-get clean
+RUN apt-get update && apt-get install -y php-cli php-mbstring && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
What if we make this rm -rf lists improvement first, to reduce the changes to this PR?

Reply:
I can do this, this rm is simply best practice for docker with debian based distros

Reply:
see #4711